enterprise/api/v1/views.py

```diff
@@ -1203,7 +1241,7 @@ def post(self, request):
             )
 
 
-class TableauAuthViewSet(generics.GenericAPIView):
+class TableauAuthView(generics.GenericAPIView):
```

Comment:
Again, why name change? Rebase issue?

Reply:
Yeah, as mentioned above, making the name accurate.